dotnetv3/README.md

```diff
@@ -40,84 +49,17 @@ In general, follow these steps:
    and a **.csproj** file.
 4. Run the project using the ```dotnet run``` command.
 
-## Running the unit tests
-
-All of the code example projects have a companion unit test,
-where the name of the unit test project is the same as the tested project,
-with a **Test** suffix.
-
-We use [Xunit](https://xunit.net/) as the framework for our unit tests and
-in most cases use [moq4](https://github.com/moq/moq4) to create unit tests with mocked objects.
-You can create an Xunit project and install the **moq** Nuget package in that project with
-the following commands:
-
-```
-dotnet new xunit -o MyTestProject
-cd MyTestProject
-dotnet add package moq
-```
+## Tests
+⚠️ Running the tests might result in charges to your AWS account.
 
-A typical unit test looks something like the following,
-which tests a call to **CreateTableAsync** in the
-**CreateTable** project:
+Most service folders also include a test project and either integration tests, unit tests, or both. 
+To run all the tests, navigate to the folder that contains the test project and then issue the following command:
 
 ```
-using System.Net;
-using System.Threading;
-using System.Threading.Tasks;
-
-using Amazon.DynamoDBv2;
-using Amazon.DynamoDBv2.Model;
-
-using Moq;
-
-using Xunit;
-
-namespace DynamoDBCRUD
-{
-    public class UnitTest1
-    {
-        private static readonly string _tableName = "testtable";
-
-        private IAmazonDynamoDB CreateMockDynamoDBClient()
-        {
-            var mockDynamoDBClient = new Mock<IAmazonDynamoDB>();
-
-            mockDynamoDBClient.Setup(client => client.CreateTableAsync(
-                It.IsAny<CreateTableRequest>(),
-                It.IsAny<CancellationToken>()))
-                .Callback<CreateTableRequest, CancellationToken>((request, token) =>
-                {
-                    if (!string.IsNullOrEmpty(_tableName))
-                    {
-                        bool areEqual = _tableName == request.TableName;
-                        Assert.True(areEqual, "The provided table name is not the one used to create the table");
-                    }
-                })
-                .Returns((CreateTableRequest r, CancellationToken token) =>
-                {
-                    return Task.FromResult(new CreateTableResponse { HttpStatusCode = HttpStatusCode.OK });
-                });
-
-            return mockDynamoDBClient.Object;
-        }
-
-        [Fact]
-        public async void CheckCreateTable()
-        {
-            IAmazonDynamoDB client = CreateMockDynamoDBClient();
-
-            var result = await CreateTable.MakeTableAsync(client, _tableName);
-
-            bool ok = result.HttpStatusCode == HttpStatusCode.OK;
-            Assert.True(ok, "Could NOT create table " + _tableName);
-        }
-    }
-}
+dotnet test
 ```
 
-To run this test,
-navigate to the **CreateTableTest** folder and run:
+Alternatively, you can open the example solution and use the Visual Studio Test Runner to run the tests.
```

Comment:
GH won't let me highlight the line I want, but the `dotnet test` block right after this line appears to be extraneous?

Reply:
Ah, good catch. I decided to remove the whole section on running from the test runner, because we have found it to be problematic with ordered tests anyway.